project1.css

```diff
@@ -1,46 +1,218 @@
-.navbar {
-	background-color: #7a7676;
+nav {
+	background-color: #fc7b03;
 	display: flex;
+    top: 0;
+    left: 0;
+    width: 100%;
+    padding: 5px 0;	
+	position: fixed;
+	z-index: 2;
+	text-align: center;
 	justify-content: center;
 }
 
-.nav-button {
-    display: inline-block;
+nav a {
     border: none;
-    padding: 1rem 2rem;
-    margin: 0;
+    padding: 1rem 1rem;
+	width: 20%;
     text-decoration: none;
-    background: #7a7676;
-    color: #ffffff;
-    font-family: Arial;
-    font-size: 1rem;
+    background: #fc7b03;
+    color: #000000;
+    font-family: "Andale Mono";
+	font-weight: bold;
     cursor: pointer;
-    text-align: center;
-    transition: background 250ms ease-in-out, 
-                transform 150ms ease;
-    -webkit-appearance: none;
-    -moz-appearance: none;
 }
 
-.nav-button:hover,
-.nav-button:focus {
-    background: #fc7b03;
+nav a:hover,
+nav a:focus {
+    background: #db6a00;
+}
+
+.intro-section {
+	font-family: "Andale Mono";
+	justify-content: center;
+	align-items: center;
+	padding-left: 200px;
+	color: #ffffff;
+	background-color: #fc7b03;
+	background-image: linear-gradient(to bottom right, #fc7b03, #000000);
+	position: relative;
+	min-height: 100vh;	
 }
 
-.nav-button:focus {
-    outline: 1px solid #fff;
-    outline-offset: -4px;
+.intro-section div {
+	text-align: center;
+	padding-top: 100px;
 }
 
-.nav-button:active {
-    transform: scale(0.99);
+.intro-section img {
+	position: absolute;
+	bottom: 0;
+	left: 10px;
+	height: 20vw;
 }
 
-.intro-page {
-	font-family: Arial;
+h1 {
+	font-size: 6rem;
+}
+
+header h2 {
+	font-size: 4rem;
+}
+
+.about-section {
+	background-color: #000000;
+	min-height: 100vh;
+	font-family: Andale Mono;
+	padding-left: 200px;
+	display: flex;
+	color: #fc7b03;
+}
+
+.about-section h2 {
+	font-size: 5rem;
+}
+
+.about-section div {
+	font-size: 1.5rem;
+	width: 60%;
+}
+
+.about-image {
+	padding-top: 150px;
 	padding-left: 40px;
-	color: white;
+	flex-direction: column;
+}
+
+.about-section a {
+	color: inherit;
+}
+
+.portfolio-section {
 	background-color: #fc7b03;
-	background-image: linear-gradient(to bottom right, #fc7b03, #000000);
+	font-family: "Andale Mono";
+	font-weight: bold;
+	padding-left: 200px;
+	color: #000000;
+	display: flex;
+	flex-direction: column;
+	gap: 20px;
+	min-height: 100vh;
+}
+
+.portfolio-section h2 {
+	font-size: 5rem;
+}
+	
+.project-containers {
+	box-shadow: 3px 3px 2.5px #000000;
+	border-radius: 5px;
+	border-style: ridge;
+	width: 65%;
+	padding: 20px;
+	font-size: 1.5rem;
+	display: flex;
+	gap: 20px;
+
+}
+
+.project-images img {
+	height: 150px;
+	width: 150px;
```

Comment:
this will stretch images that are not square - you might want to look into `object-fit` (optional)